src/routes/videoPlaybackProxy.ts

```diff
@@ -145,6 +145,9 @@ videoPlaybackProxy.get("/", async (c) => {
     // into chunks and stream the response back to the client to avoid rate limiting
     const { readable, writable } = new TransformStream();
     const stream = new StreamingApi(writable, readable);
+    stream.onAbort(() => {
```

Comment:
Please create another PR for such additional messages.

Same for https://github.com/iv-org/invidious-companion/pull/253/files#diff-47a89f5a87c3e5f5e6ec27049f51989e1f1a44e0c09cdc50dd052bfb9d249be8R194-R204